source/sonata_config.rst

```diff
@@ -82,6 +82,8 @@ They can be found also under :ref:`populations <population_config_dict>` where i
                                                     This path is used in conjonction with the spine_morphology property (see :doc:`sonata_tech`) to find the spine_morphology.
                                                     By default, the concatenation of the spine_morphologies_dir + spine_morphology_property + ".h5" extension.
    provenance                      Optional         Dictionary containing provenance metadata (e.g. bioname).
+   spatial_index_dir               Optional         Path to the directory containing the spatial index files.
+                                                    Only for `biophysical` node populations and `chemical` edge populations.
```

Comment:
The new property should be added to the nodes and edges populations properties described below, instead of the global components.